MODULE.bazel

```diff
@@ -1,6 +1,6 @@
 module(
     name = "rules_bzlmodrio_jdk",
-    version = "17.0.12-7",
+    version = "17.0.12-8",
```

Comment:
`17.0.12-7` refers to the toolchain version. When I do stuff like this I do `.bcr1`